### ISSUE TITLE
Remove Resize large images original source.

### ISSUE THREAD

**Issue by selul** [Codeinwp/optimole-wp#675]
### What problem does this address?

Right now, we have two features that somehow share the functionalities among them up to a point. For the sake of simplicity, I will call them: 

- **RLS** - Resize large images from the original source.
- **LMD** - Limit Image Dimensions with max width/height

On the PHP Side, RLS is doing the resize to those boundaries if the image size is [known](https://github.com/Codeinwp/optimole-wp/blob/master/inc/url_replacer.php#L254-L260) ( not auto ) while the LMD is [covering](https://github.com/Codeinwp/optimole-wp/blob/master/inc/url_replacer.php#L229-L234) the auto case as well, by forcing those boundaries by default to the auto images as well. 

Moreover, the LMD is also covering the Autoscalling feature, which the RLS is not, since is doing only the backend stuff, which is pre-browser resizing or lazyload. 

### What is your proposed solution?

I suggest remove RLS together, don't merge any value, and keep the LMD as it is. 

The impact of this is that old users who were having RLS will no longer use it, and to accommodate this; I would suggest we turn on LMD as well for legacy users. 

@Codeinwp/optimole do you see any case that could not be covered by this? Like a use case or similar that would be affected by the change. 

Keeping you in the loop @Codeinwp/design-team 

### Will this feature require documentation? (Optional)

Yes, it requires documentation.

**Comment by mghenciu** [comment]
With this changes, will the user needs [mentioned by Kush here](https://vertistudio.slack.com/archives/CPH2KCMN1/p1695331833991169) will still be meet? or there is just a confusion on how the feature works

**Comment by abaicus** [comment]
@selul, we should be fine removing the RLS setting and its impact on the code. I just have one concern:

If we turn LMD for all users, keeping the default values (1920x1080), couldn't this potentially impact the quality displayed for old users that had a different value (larger) for the RLS setting? According to the previous value, the limit was 2000x1500 for large images, actually. I get that the difference won't be that much for the defaults (1920 vs 2000 and 1500 vs 1080), but it can still affect sites that did change this setting for purposes of displaying higher-quality images.

**Comment by selul** [comment]
Yes, however, I think it should be acceptable.

**Comment by mghenciu** [comment]
For this one, I don't think that there's any design work involved. Ping us if needed.
Probably would make sense to do another UX Writing check after we have the RLS removed and the new text descriptions.

**Comment by abaicus** [comment]
@selul One more question regarding this. In the case of images having `auto` width/height, or both, should we simply set the `w:`/`h:` to the value of the limit or still keep `auto`? 

Right now, there's no way of telling the proper size in those cases without knowing the image's aspect ratio. Previously, both settings kept `auto` sizes for these parameters, as they only checked a strict comparison with the `auto` string cast as int (which is `0`), in which case, the URLs used `auto` for w/h.

I would keep the same approach as we did until now in these cases (i.e. keeping `auto`)

Thank you! 🙏🏻
